src/main/java/www/purple/mixxy/views/layout/defaultLayout.ftl.html

```diff
@@ -1,47 +1,50 @@
 <#macro myLayout title="Layout example">
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <script src="/assets/webjars/jquery/1.9.0/jquery.js"></script>
-    <meta charset="utf-8">
-    <title>${title}</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="">
-    <meta name="author" content="">
-
-    <!-- Le styles -->
-    <link href="/assets/webjars/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet">
-    <!-- Optional theme -->
-      <link rel="stylesheet" href="/assets/webjars/Semantic-UI/2.1.8/semantic.min.css">
-      <link rel="stylesheet" href="/assets/mixxy.css">
-    
-
-    <!-- Latest compiled and minified JavaScript -->
-    
-    <style type="text/css">
-      body {
-        padding-top: 60px;
-        padding-bottom: 40px;
-      }
-    </style>
-
-    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
-    <!--[if lt IE 9]>
-      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
-  </head>
-
-
-  <body>
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <script src="/assets/webjars/jquery/1.9.0/jquery.js"></script>
+        <meta charset="utf-8">
+        <title>${title}</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="">
```

Comment:
Some description for Mixxy would be good here.  It's good for SEO.  Maybe something like "Mixxy is a collaborative art platform where users can build upon each other's works".